File: SFA-API/Domain/Area.cs
using System.Collections.Generic;

namespace Domain
{
    public class Area
    {
        public int Id { get; set; }
        public int DistrictId { get; set; }
        public string Name { get; set; }
        public string AreaCode { get; set; }
        public virtual District District { get; set; }
        public virtual ICollection<Route> Routes { get; set; }
    }
}

File: SFA-Client/src/app/_models/area.ts
export class Area {
  id: number;
  districtId: number;
  provinceId: number;
  name: string;
  areaCode: string;

  constructor(area) {
    this.id = area.id;
    this.districtId = area.districtId;
    this.provinceId = area.provinceId;
    this.name = area.name;
    this.areaCode = area.areaCode;
  }
}
